modules/@apostrophecms/ui/ui/apos/mixins/AposPublishMixin.js

```diff
@@ -36,7 +36,11 @@ export default {
           dismiss: true,
           icon: 'check-all-icon'
         });
-        return true;
+        apos.bus.$emit('content-changed', {
```

Comment:
This file should have an `emits` array with this one event.

Reply:
Bus events are not vue events so they should not be on the "emits" array.